tools/plugin-toolkit/src/_base/cli/validators.ts

```diff
@@ -34,3 +37,54 @@ export function hostnameValidatorFactory(isBear: boolean) {
         }
     };
 }
+
+export function pluginNameValidator(value: string): boolean | string {
+    if (!value.match(/^[a-zA-Z0-9_\-@/]*$/)) {
+        return "Invalid plugin name. Use only alphanumerical characters, underscores and dashes.";
```

Comment:
Would not be better to return Error instead of string value only? There are functions returning error as well as strings and we should have this unified in my opinion. Applies for all functions returning string error in this file.

Reply:
This is the API of the inquirer, the error must be a string. It is not a traditional error, it is more of  a validaion result.

Reply:
indeed, the signature for validation functions is based on what inquirer needs so that single validation can be used in there & when validating inputs coming directly from command line